jvm-packages/pom.xml

```diff
@@ -37,6 +37,7 @@
         <spark.version>2.4.3</spark.version>
         <scala.version>2.12.8</scala.version>
         <scala.binary.version>2.12</scala.binary.version>
+        <rabit.mock>OFF</rabit.mock>
```

Comment:
it's very strange or ungraceful that we have to leak a switch which is used in running a test in build files like pom.xml.....

Reply:
What would be other options on top of your mind?

Reply:
replace with env setting instead of property